bin/mail/2016-08-09-hoc-latam/send-real-emails

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+#3661
```

Comment:
Oh is 3661 the # that it sends to, and you're updating to 9055? Maybe add more to the comment `# will send to 9,055 emails after unsubscribes`.

Does limit 4000 mean the size of a batch then?